gluoncv/utils/metrics/tracking.py

```diff
@@ -3,6 +3,36 @@
 from colorama import Style, Fore
```

Comment:
I didn't notice this last time, colorama is not a standard library. Maybe we need to write a `try_import_colorama` function. Let's discuss first.

Reply:
https://github.com/dmlc/gluon-cv/blob/master/gluoncv/utils/filesystem.py#L43